components/Fields/CheckboxGroup.js

```diff
@@ -0,0 +1,98 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Checkbox from './Checkbox';
+import { asOptionObject, getValue } from './utils/options';
+
+class CheckboxGroup extends PureComponent {
+  static propTypes = {
+    options: PropTypes.array,
+    className: PropTypes.string,
+    label: PropTypes.string,
+    fieldLabel: PropTypes.string,
+    input: PropTypes.object.isRequired,
+    optionComponent: PropTypes.component,
```

Comment:
`PropTypes.component` is undefined; could use `.node` or `oneOfType[...]` instead